### ISSUE TITLE
NoValueAvailableException is inaccessible due to its protection level

### ISSUE THREAD

**Issue by lawrence-laz** [ClosedXML/ClosedXML#896]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

0.92.1

**What is the current behavior?**

[NoValueAvailableException ](https://github.com/ClosedXML/ClosedXML/blob/develop/ClosedXML/Excel/CalcEngine/Exceptions/NoValueAvailableException.cs) is internal, therefore it is inaccessible and cannot be used in try/catch.

**What is the expected behavior or new feature?**

[NoValueAvailableException ](https://github.com/ClosedXML/ClosedXML/blob/develop/ClosedXML/Excel/CalcEngine/Exceptions/NoValueAvailableException.cs) should be public so it could be handled in user code.

**Did this work in previous versions of our tool?  Which versions?**

No

## Reproducibility

**Code to reproduce problem:**
```c#
public void Main()
{
    try
    {
        return;
    }
    catch(ClosedXML.Excel.CalcEngine.Exceptions.NoValueAvailableException e)
    {
    }
}
```

**Comment by igitur** [comment]
Damn, this was a pretty serious oversight on my part.

**Comment by igitur** [comment]
Fixed in #897
